rfcs/react-components/non-core-components.md

```diff
@@ -0,0 +1,388 @@
+- 👍 Easier discoverability for OSS users and internal partners
+
+##### Cons
+
+- 👎 Not clear that there is a difference between non-core components and core components/utilities
+- 👎 Harder for us to tell the difference between non-core and core
+
+#### New package scope
+
+Creates a new package scope, `@fluentui-banana/` (name pending) for all components/packages that are non-core.
+
+##### Pros
+
+- 👍 Very clear differentiation between core and non-core
+- 👍 Creates a clear boundary for FUI team and partners to collaborate
+- 👍 Clear path to upgrade/downgrade packages between core and non-core (i.e. experiments/compat)
+
+##### Cons
+
+- 👎 Need to maintain a new NPM organization and credentials
+- 👎 Initially could cause some confusion if we don't communicate this well
+
+### How users should consume components
+
+We should keep the same architecture we currently have for our core components: one package per component. What should
+be decided here is how users consume these components.
+
+#### Suite packages
+
+Structure non-core components into a new suite package similar to `@fluentui/react-components`.
+
+##### Pros
+
+- 👍 Similar to core components
+- 👍 Easy to update when using multiple components
+
+##### Cons
+
+- 👎 Needs to follow the breaking change cadence of each components
+- 👎 Slows down breaking changes for all components
+
+#### Individual packages
+
+Users should install each component package individually.
+
+##### Pros
+
+- 👍 Easy to pick and choose components that might not always be relevant
+- 👍 Allows more flexible breaking change cadence
+
+##### Cons
+
+- 👎 Easier to install duplicate versions
+- 👎 We might be encouraged to break too often
+
+### How to depend on core components
+
+#### Depend directly on components packages
+
+Each non-core component should depend on the individual core components
+
+##### Pros
+
+- 👍 Users will always install the required core components
+
+##### Cons
+
+- 👎 Consumes core packages in a way we don't recommend other users
+- 👎 Dependency graph could be hard to manage/track
+- 👎 Internal API might become public
+
+#### Depend on suite package is peer dependency
+
+#### Pros
+
+- 👍 No unnecessary dependencies - we expect users to have core suite package installed
+- 👍 Other examples in OSS - Material UI/React all recommend peer dependencies
+- 👍 Easy for users to consume as long as core does not break
+- 👍 We test how 'complete' our public API is
+
+#### Cons
+
+- 👎 V10 upgrade could be unpleasant - need to check what range is supported for each package
+- 👎 More of our internal API becomes public
+
+### Documentation
+
+Fluent UI production components should always be documented regardless of whether they are core/non-core components.
+This section describes the ways to document non-core components.
+
+#### Current docsite
+
+Create a new section in our docsite explicitly for non-core components alongside core components.
+
+##### Pros
```

Comment:
from experience/feedback it's best DX/UX to have all FUI family under one roof.  

I'd avoid going into more technical details within this RFC convo

Reply:
I've scaled down the proposals to monorepo and separate repo, let's discuss offline with a smaller audience on the technical details